doc/api/cli.md

```diff
@@ -154,53 +208,80 @@ Note: v8 options allow words to be separated by both dashes (`-`) or underscores
 For example, `--stack-trace-limit` is equivalent to `--stack_trace_limit`.
 
 ### `--tls-cipher-list=list`
+<!-- YAML
+added: v4.0.0
```

Comment:
Yes, this originally landed in v0.12 and did not come into any of the io.js streams. It landed in v4 as part of the convergence.